build.sbt

```diff
@@ -4,7 +4,7 @@ name := "riak-scala-client"
 
 version := "0.9.5"
 
-scalaVersion := "2.11.5"
+scalaVersion := "2.11.7"
 
 crossScalaVersions := Seq("2.11.5", "2.10.4")
```

Comment:
Scala versions have gone up too (2.11.7 and 2.10.5 AFAIK). Might as well update those.